services/resource-usage-tracker/requirements/_base.txt

```diff
@@ -806,7 +832,7 @@ types-aiobotocore==2.19.0
     # via -r requirements/../../../packages/aws-library/requirements/_base.in
 types-aiobotocore-ec2==2.19.0
     # via types-aiobotocore
-types-aiobotocore-s3==2.19.0
+types-aiobotocore-s3==2.19.0.post1
```

Comment:
No `_base.txt` in a service is supposed to change. And this one already has pydantic-settings<2.7.1

Reply:
This has nothing to do with the `pydnatic-settings`.

The `types-aiobotocore-s3` fixed a typo where they had `Request` repeated twice in a name. If I update the underlying package inside `aws-library` without this the service will not start since it will not find the new class.

This is not ideal but required.